vtexdocs/devportal
# Fix/sidebar structure

#### What is the purpose of this pull request?

Update the sidebar structure to the new structure defined on [JSON master](https://www.notion.so/vtexhandbook/Estrutura-do-json-master-1472c3ec0a05453391cdb73c0b698870) and update de sidebar section with the filter component.

#### What problem is this solving?

[Documented Here :book: (in building) ](https://www.notion.so/vtexhandbook/Atualizar-a-sidebar-5b07fc55a0ae4fefa695b260a3c6de63)

The sidebar structure is outdated and the API Reference section (in the sidebar) needs to have a filter component.

#### How should this be manually tested?

Check the code (typos or possible improvement), after go to this page and check:
- If the sidebar appearance looks okay following [the Figma](https://www.figma.com/file/Lx6sXdrw9KEvtSEKWttmv8/Developer-Portal?node-id=6005%3A356800).
- Check if the API Reference page has the sidebar.
- Check if the API Reference section contains the filter component.	
  - Check the appearance and behavior of this component compared with [this flow in Figma](https://www.figma.com/proto/Lx6sXdrw9KEvtSEKWttmv8/Developer-Portal?node-id=6309%3A379335&scaling=min-zoom&page-id=6005%3A615480&starting-point-node-id=6005%3A615785).
  - Try to play with the filter if it works.
  - Make sure the "Clear all" option appears when two or more filters are selected.

#### Screenshots or example usage

https://user-images.githubusercontent.com/42784961/183113433-27cb2b3c-011b-436f-92ff-00c1c3ad5df5.mp4

#### Types of changes
